src/AspNet.Security.OAuth.Introspection/Events/AccessTokenReceivedContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OAuth.Introspection {
+    public class AccessTokenReceivedContext : BaseIntrospectionContext {
```

Comment:
Please consider adding some XML documentation love for all the public stuff :smile: